os/drivers/lcd/lcd_dev.c

```diff
@@ -0,0 +1,395 @@
+/****************************************************************************
```

Comment:
Simply, you can use .tizenrt/toos/cformatter.sh
But above tool can't distingush between pointer and multiplication.
so It will change code like below. so it should be updated manually.
int *a -> int * a